node/core/pvf/tests/it/main.rs

```diff
@@ -97,10 +100,14 @@ async fn terminates_on_timeout() {
 		Err(ValidationError::InvalidCandidate(InvalidCandidate::HardTimeout)) => {},
 		r => panic!("{:?}", r),
 	}
+
+	let duration = std::time::Instant::now().duration_since(start);
+	assert!(duration >= TEST_EXECUTION_TIMEOUT);
+	assert!(duration < TEST_EXECUTION_TIMEOUT * JOB_TIMEOUT_WALL_CLOCK_FACTOR);
```

Comment:
```suggestion
```

I know that you added this "for me"? I think we don't need it.

Reply:
You don't think it's useful at all? I think this is the most direct test we have that the CPU thread works as intended.